controllers/netbot.js

```diff
@@ -821,8 +804,13 @@ class netBot extends ControllerBot {
             this.tx(this.primarygid, "200", msg)
             sysManager.clearBranchChangeFlag()            
           }
-
-        } else {
+        }
+        else if (upgradeInfo.upgraded) {
+          let msg = `Firewalla is upgraded to ${upgradeInfo.to}`;
```

Comment:
enhancement: use locale (en.json, zh.json) to localize upgrade message

Reply:
#1279